tests/copyoffload/test_copyoffload_migration.py

```
@@ -4639,29 +4743,93 @@ def test_migrate_vms_simultaneously(
+                            f"Plan {self.plan_resource_2.name} failed. \nstatus:\n\t{self.plan_resource_2.instance}"
+                        )
+
+                # Break when both plans have completed
+                if len(completed_plans) == 2:
+                    break
+
+        except TimeoutExpiredError:
+            raise MigrationPlanExecError(
+                f"One or both migrations failed to complete within timeout. "
+                f"Completed: {completed_plans}, Expected: {{{self.plan_resource_1.name}, {self.plan_resource_2.name}}}"
+            )
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)